job.js

```diff
@@ -30,6 +49,10 @@ class BigQueryJob {
         this.costThresholdInGB = opts.costThresholdInGB != null ? opts.costThresholdInGB : COST_THRESHOLD_IN_GB;
         this.costPerTB = opts.costPerTB != null ? opts.costPerTB : COST_PER_TB;
 
+        this.destinationTableConfig = opts.destinationTableConfig;
+        this.writeDisposition = opts.writeDisposition;
+        this.avoidReturningResults = opts.avoidReturningResults || false;
```

Comment:
I prefer avoid the "negative". I would use `this.shouldQueryResults` and default as `true`